### ISSUE TITLE
Cannot change any css value after editing color with firepicker

### ISSUE THREAD

**Issue by dedou** [thedarkone/firepicker#12]
Hi,
I am on Firefox 5.0 / Firebug 1.7.3
In the HTML Tab , after editing a color with firepicker, I cant change any css value until reload the page  ...

the value box appear empty, you can enter a new value, but it doesnt change the original value after validate.

Is there a solution ?

Thanks
Didier

**Comment by thedarkone** [comment]
Hey,

I'm unable to reproduce this. Could perhaps try reinstalling firebug/firepicker or disabling some other plugins to figure out if there's some other conflict?

**Comment by xxvii** [comment]
Hi,
exactly the same issue here.
firefox 5.0.1 (and previous versions), firebug 1.8.0 and 1.7.3, firepicker 1.3.6 on windows 7 ultimate 64 bit.
The problems persists with all other extensions deactivated.
The problem happened after the upgrade of firebug from version 1.6 to version 1.7.

To restore the functionality you have to close firebug with his rightmost red button (the bug icon must be grey) and open it again. It is not necessary to reload the page, althought it works as well.

With firebug 1.6.2, firepicker 1.3.6 and all other extensions activated all is working correctly

thank you very much!

**Comment by thedarkone** [comment]
Hey,

sorry for taking so long to look into this. I've just tested this with Firebug 1.7.3 and 1.8.1b2 to no vain. This might be a Windows issue as I'm on Mac. I'll try to get mine hands on a Windows box to play with it.

**Comment by xxvii** [comment]
Thank you very much thedarkone, I use firepicker for work everyday and it's really one of the most useful extensions, it's a pity that it is buggy on some computers since the upgrade to firebug 1.7...

**Comment by dedou** [comment]
Hi,
Just a word to confirm i'm on windows to :
Windows 7 pro 64 / Firefox 6.0 / Firebug 1.8.1

Thanks !

**Comment by ghost** [comment]
The same for me here. Tried on Windows 7 and Xp wih Firefox >= 4 and a couple of the latest versions of firebug (1.6, 1.7,...). No avail. Can't edit any CSS after picking a color. Need to refresh page. Very sad.

Andrew

**Comment by xxvii** [comment]
shinystar, you don't need to refresh the page in order to continue your editing in firebug!

You just need to close (with the red "close" button, NOT pressing F12) and open firebug again, and keep editing without losing your previous changes!

**Comment by ghost** [comment]
Great, I'll give that a try later!

Thanks!
Andrew

**Comment by mrtorrent** [comment]
Same issue here. Makes Firebug with Firepicker unusable, unfortunately.
- Firepicker 1.3.6
- Firebug 1.8.4
- Firefox 7.0.1

**Comment by thedarkone** [comment]
Hey guys,

this is really frustrating, as I'm unable to repro this on a windows machine :(
- Windows 7 32-bit
- Firefox 7.0.1
- Firebug 1.8.4
- Firepicker 1.3.6

Maybe I need to be running the 64-bit version of windows...

**Comment by xxvii** [comment]
you could try...though it's very strange you don't have the same problem...

I have windows ultimate 64bit indeed, and I keep having the same problem also in firefox 8 (with all the addons in their latest versions)

As I wrote earlier, the problem happened after the upgrade of firebug from version 1.6 to version 1.7, maybe you should try to contact the firebug developers...

thanks again! :)

**Comment by thedarkone** [comment]
Tested this on Windows x64 machine FF 8.0.1, FB 1.8.4b1, Firepicker 1.3.6, still no luck :(. Do you get any errors in the Firefox´s Error Console Ctrl+Shift+J?

**Comment by xxvii** [comment]
well, in this very page, as soon as the page finish to load, I found some of these (and similar others always related to css):

Avviso: Errore durante l'interpretazione di 'display'.  Dichiarazione tralasciata.
File sorgente: https://github.com/thedarkone/firepicker/issues/12#issuecomment-3276597
Riga: 0

and these:
Avviso: L'utilizzo di getAttributeNodeNS() è deprecato. Al suo posto utilizzare getAttributeNS().
File sorgente: https://github.com/thedarkone/firepicker/issues/12#issuecomment-3276597
Riga: 0

Avviso: L'utilizzo di getAttributeNode() è deprecato. Al suo posto utilizzare getAttribute().
File sorgente: https://github.com/thedarkone/firepicker/issues/12#issuecomment-3276597
Riga: 0

Avviso: L'utilizzo di nodeName su attributi è deprecato. Al suo posto utilizzare name.
File sorgente: https://github.com/thedarkone/firepicker/issues/12#issuecomment-3276597
Riga: 0

Avviso: L'utilizzo di nodeValue su attributi è deprecato. Al suo posto utilizzare value.
File sorgente: https://github.com/thedarkone/firepicker/issues/12#issuecomment-3276597
Riga: 0

sorry if they are in italian, I'm sure that google translate will give you the respective english ones ;)

**Comment by dedou** [comment]
Hi,
Windows 7 x64 - Firefox 9.0.1 Firebug 1.8.4
Perhaps this will help you
After editing a color with firepicker, I cannot change any other css value like `height` for exemple using the numeric key, but if I use up or down arrow I got this error :
Erreur : uncaught exception: [Exception... "Component returned failure code: 0x80004005 (NS_ERROR_FAILURE) [nsIDOMHTMLInputElement.setSelectionRange]"  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "JS frame :: resource://firebug_rjs/editor/editor.js :: <TOP_LEVEL> :: line 766"  data: no]

best regards

**Comment by thedarkone** [comment]
Tested on another Win x64 SP1 box, still no luck.

Have you tried Firebug 1.9.x?

**Comment by xxvii** [comment]
yes, nothing changes with the lastest beta

**Comment by thedarkone** [comment]
Can you try running Firepicker and Firebug with [FBTrace](http://getfirebug.com/wiki/index.php/FBTrace) enabled and send me the trace produced?

**Comment by xxvii** [comment]
I installed the latest beta version of FBTrace, restarted firefox, here's the log: http://www.mediafire.com/?2c4smmn3dzztn2s

When I changed a color with firepicker, nothing happened in fbtrace...I don't know if it is supposed to do something or not

**Comment by thedarkone** [comment]
It means nothing, I was just hoping something might show up ;(.

Alright, since we know the problem shows up somewhere between Firebug 1.6.2 and 1.7.0 I tried to look at the diffs and see if anything pops up. The only main addition to the editor's code seems to be a more evolved handling of text selection added [here](http://code.google.com/p/fbug/source/detail?r=8902&path=/branches/firebug1.7/content/firebug/editor.js). Can you see if removing it from Firebug fixes the problem?

Here are the two FB 1.9 test xpi archives: [one](http://www.mediafire.com/?5tugfk2pysfuy33), [two](http://www.mediafire.com/?jgxpz94f088hw2e). Please test both of them.

**Comment by xxvii** [comment]
I tested both archives but nothing changed.

I tried to disable all the extensions beside fb and fp: nothing

I tried to uninstall all the extensions beside fb and fp: nothing

then I created a new clean firefox profile and I tried the last "normal" version of firebug and firepicker, and it worked correctly

I reinstalled ALL the extensions I had (included fb and fp), restored history, cookies, password, certificates etc. that I backupped with mozbackup, BUT NOT the general settings NOR the "unknown" files belonging to the extensions and...IT WORKED!

Maybe the problem is in one of the general settings, but I have no idea of which one :(
